CHANGELOG.md

```diff
@@ -1,22 +1,37 @@
-## 1.1.0 - 2024-06-25
-[GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
-## 1.0.9 - 2024-02-12
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.1.0] - 2024-06-25
```

Comment:
It looks like not all tags are here, we will need to add missing tags and list the changes